### ISSUE TITLE
Update make run command

### ISSUE THREAD

**Issue by cypherai** [OpenMined/PySyft#186]
Following PR #175

**Comment by MtDersvan** [comment]
The Docker image is updated without jupyter dependency, but make run has not been updated yet, thus people who run Usage:Start: `make run` will encounter something like this:
```
docker: Error response from daemon: oci runtime error: container_linux.go:262: starting container process caused "exec: \"jupyter\": executable file not found in $PATH".
make: *** [run] Error 127
```

**Comment by digoreis** [comment]
Hey, 

I created a Dockerfile to development local. And a command : 
`make custom  docker="nameofcontainer" `

More details in this [Pull Request](https://github.com/OpenMined/PySyft/pull/236)

**Comment by baldassarreFe** [comment]
Closed with #270, please check if it works as expected
